gen_secrets_for_airtable.py

```diff
@@ -0,0 +1,144 @@
+from datetime import datetime
+import hashlib
+import logging
+import os
+
+from pywell.secrets_manager import get_secret
+from pywell.entry_points import run_from_cli
+
+# MUST be above all parsons imports to avoid full parsons dependencies
+os.environ["PARSONS_SKIP_IMPORT_ALL"] = "1"
```

Comment:
this flag won't do anything, was removed from parsons a while ago. all you need to do is set the env var `PARSONS_LIMITED_DEPENDENCIES=1` before installing